docs/blockchain/blockchain-primitives.mdx

```diff
@@ -31,15 +31,7 @@ The easiest way to see the blocks being mined is with the
 [Helium Mobile App](https://helium.com/app) also shows block details. Developers can also use the
 Helium blockchain API to access all blockchain transactions and metadata.
 
-### Epochs
-
-An `epoch` is the target time period for which a given group of Validators is elected to serve as
-the consensus group. The target time for an epoch is currently `30 blocks`, as defined in the
-`election_interval` chain variable. Approximately every 30 blocks mined marks the passing of an
-epoch, after which a new group of Validators is elected to form the next consensus group. Mining
-rewards are distributed per epoch (as opposed to per block in most blockchain-based systems). At the
-conclusion of each epoch, the consensus group will distribute all the $HNT produced in that block
-via the `rewards` transaction.
+<!--TODO: document epochs on soland and in helium rewards -->
 
 ### Transactions
```

Comment:
This whole section could probably go. No more htlc, state channels, etc.

Reply:
trimming